ci-operator/step-registry/cucushift/hypershift-extended/external-oidc/health-check/cucushift-hypershift-extended-external-oidc-health-check-commands.sh

```diff
@@ -35,17 +35,18 @@ if [[ -z "$hc_authentication_cluster_spec_oidcproviders" ]]; then
     exit 1
 fi
 
+# TODO: Uncomment this when OCPBUGS-57736 is fixed.
 # Check special fields in authentication.config/cluster
-if oc get featuregate cluster -o=jsonpath='{.status.featureGates[*].enabled}' --kubeconfig "${SHARED_DIR}"/nested_kubeconfig | grep -q ExternalOIDCWithUIDAndExtraClaimMappings; then
-    # Ensure the extra and uid fields exist and are not empty
-    if grep -q '"extra":\[{"key".*"uid":{"' <<< "$hc_authentication_cluster_spec_oidcproviders"; then
-        echo "External OIDC uid and extra settings are synced into the hosted cluster"
-    else
-        echo "$hc_authentication_cluster_spec_oidcproviders"
-        echo "External OIDC uid and extra settings are not synced into the hosted cluster!"
-        exit 1
-    fi
-fi
+# if oc get featuregate cluster -o=jsonpath='{.status.featureGates[*].enabled}' --kubeconfig "${SHARED_DIR}"/nested_kubeconfig | grep -q ExternalOIDCWithUIDAndExtraClaimMappings; then
+#     # Ensure the extra and uid fields exist and are not empty
+#     if grep -q '"extra":\[{"key".*"uid":{"' <<< "$hc_authentication_cluster_spec_oidcproviders"; then
+#         echo "External OIDC uid and extra settings are synced into the hosted cluster"
+#     else
+#         echo "$hc_authentication_cluster_spec_oidcproviders"
+#         echo "External OIDC uid and extra settings are not synced into the hosted cluster!"
+#         exit 1
+#     fi
+# fi
```

Comment:
Since the console is fixed now, the commenting out should be needed now.

Reply:
Hmm. We also want to run this with 4.19 and it's not fixed there yet.
I can make this stuff conditional for 4.19. @xingxingxia  WDYT?